components/new-homepage/TheFutureIsCollectiveSection.js

```diff
@@ -0,0 +1,129 @@
+import React, { Fragment, useState } from 'react';
+import { RightArrow } from '@styled-icons/boxicons-regular/RightArrow';
+import Link from 'next/link';
```

Comment:
Never use `next/Link` directly, use the `Link` component. Same goes with `components/new-homepage/RaiseMoneySection.js`